packages/frontend/public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
```

Comment:
The service isn't available publicly, so I'm not expecting to have any robots scraping us.